File: 1877_Minimize_Maximum_Pair_Sum_in_Array/solution.c
int comp(const void* a, const void* b) { return (*(int*)a - *(int*)b); }
int minPairSum(int* nums, int numsSize) {
    qsort(nums, numsSize, sizeof(nums[0]), comp);
    int max = 0, sum = 0;
    for (int i = 0; i < numsSize / 2; i++)
    { sum = nums[i] + nums[numsSize - 1 - i]; max = sum > max ? sum : max; }
    return max;
}

File: 1877_Minimize_Maximum_Pair_Sum_in_Array/solution.py
class Solution:
    def minPairSum(self, nums: List[int]) -> int:
        n, result = len(nums), 0
        nums.sort()
        for i in range(n // 2):
            s = nums[i] + nums[n - 1 - i]
            if s > result: result = s
        return result
